Doc/library/sys.rst

```diff
@@ -1415,6 +1419,11 @@ always available.
    ``'c_exception'``
       A C function has raised an exception.  *arg* is the C function object.
 
+   .. note::
```

Comment:
I can't reach it with a comment, but after applying the suggested changes, I recommend moving this note above the list of events to be more consistent with the other added note and avoid getting lost there. Specifically, I suggest swapping it with the `audit-event` directive, which would be consistent with how both are in `settrace` and also avoids obvious confusion (which confused me, certainly) between the *event* argument and the audit event, given the latter is currently the one mentioned immediately before "the events have the following meaning" which confusing implies it is being referred to there, rather than the events discussed in the paragraph above the `audit-event` directive.